src/navigation/actions.js

```diff
@@ -12,8 +12,7 @@ import { createRecord } from '../database/utilities';
 import { navStrings } from '../localization';
 import { ROUTES } from './constants';
 import { RootNavigator } from './RootNavigator';
-import { PrescriptionActions } from '../actions/PrescriptionActions';
-import { FinaliseActions } from '../actions/FinaliseActions';
+import { PrescriptionActions, FinaliseActions } from '../actions/index';
```

Comment:
For
`  9:1  error  Dependency cycle via ./actions:7=>../actions/FinaliseActions:16  import/no-cycle`